ads/_config.js

```diff
@@ -498,6 +498,8 @@ export const adConfig = {
 
   'ibillboard': {},
 
+  'idealmedia': {},
```

Comment:
So I noticed here you removed implementing `renderStartImplemented`. If possible, could you please implement this? It helps us serve ads as fast as possible to our users, and gives us some big performance gains.

If it's something you plan on supporting, and it's a server-side thing. I'm okay with merging this, and you can fix on your server 😄 

Let me know what you think! 👍

Reply:
Hello, at the moment we do not support this method.
We will implement it soon.
revert back to `renderStartImplemented=true` ?

Reply:
By @lannka suggestion, let's set this to `renderStartImplemented: false`, and then we can merge this in. Once you implement on the server, we can then open a new PR for `renderStartImplemented: true` 😄 👍